extensions/reactive-db2-client/deployment/src/main/java/io/quarkus/reactive/db2/client/deployment/DB2PoolBuildItem.java

```diff
@@ -7,6 +7,7 @@
 import io.quarkus.datasource.common.runtime.DataSourceUtil;
 import io.vertx.db2client.DB2Pool;
 
+@Deprecated(forRemoval = true)
```

Comment:
Do it in a subsequent PR so that we merge this one, but please add `since = "3.21"`.

Reply:
Done https://github.com/quarkusio/quarkus/pull/46461